src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,26 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => (
+  <div
+    className="CommentInfo"
+    key={comment.id}
+  >
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">
+        {comment.name}
+      </strong>
+
+      {' by '}
```

Comment:
it's better to use `span`
```suggestion
      <span> by </span>
```